docs/content/en/docs/use-cases/filename-access.md

```diff
@@ -79,16 +79,16 @@ well as appropriate filtering:
 kubectl apply -f https://raw.githubusercontent.com/cilium/tetragon/main/examples/tracingpolicy/filename_monitoring.yaml
 ```
 
-Next, you can start monitoring the events from the `xwing` pod:
+Next, you can start monitoring the events from a `file-access-pod` pod:
 
 ```bash
-kubectl logs -n kube-system -l app.kubernetes.io/name=tetragon -c export-stdout -f | tetra getevents -o compact --namespace default --pod xwing
+kubectl logs -n kube-system -l app.kubernetes.io/name=tetragon -c export-stdout -f | tetra getevents -o compact --namespace default --pod file-access-pod
```

Comment:
Not related to what you fixed but I think it would be better to use the `tetra` CLI from the container here, so that you don't need to install tetra locally and the version is on par.
```suggestion
kubectl exec -ti -n kube-system ds/tetragon -c tetragon -- tetra getevents -o compact --namespace default --pod file-access
```

Reply:
Good idea! I’ve accepted your suggestion.